src/documents/serialisers.py

```diff
@@ -810,12 +810,31 @@ class Meta:
             "sort_field",
             "sort_reverse",
             "filter_rules",
+            "page_size",
+            "display_mode",
+            "display_fields",
             "owner",
             "permissions",
             "user_can_change",
             "set_permissions",
         ]
 
+    def validate(self, attrs):
+        attrs = super().validate(attrs)
+        if "display_fields" in attrs and attrs["display_fields"] is not None:
+            for field in attrs["display_fields"]:
+                if re.sub(r"\d+", "%d", field) == SavedView.DisplayFields.CUSTOM_FIELD:
```

Comment:
What do you think about checking for `custom_field_ in field`, then the regex to match and extract, if possible?  Just thinking two regex checks here might be expensive?

Reply:
Sure, lmk what you think of this. We could probably even eliminate the regex altogether if you prefer (just remove the prefix and cast the rest to an int). Guess I just like regexs :)